src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/draftorders/upload/UploadDraftOrdersSubmittedHandlerTest.java

```diff
@@ -154,4 +166,26 @@ private String getExpectedSuggestedConfirmationBody(String caseReference) {
             + caseReference
             + "#Case%20documents).";
     }
+
+    @Test
+    void testHandleClearsUploadedDraftOrders() {
+        // Arrange
```

Comment:
Are these comments neccssary? Does this pass checkstyle?

Reply:
The build would fail if something didn't pass checkstyle.

Unit tests can be quite difficult to read and make sense of especially when a lot of test data and mock responses  are involved.
One characteristic of a well written unit test is that it is readable.
I think there is value in adding the Arrange-Act-Assert comments to divide up **_complex_** unit test methods. Although I would probably agree with you in this case - the unit test is relatively simple and they don't add much value.

Reply:
[2916286](https://github.com/hmcts/finrem-case-orchestration-service/pull/1943/commits/29162861c3385a28673c5825a82d82fa0381201e)